docs/docker_container.md

```diff
@@ -26,9 +26,9 @@ Pull an image from Docker or [RedHat Ecosystem Catalog](https://catalog.redhat.c
 
 ```bash
 docker pull openvino/model_server:latest
-
-# or, alternatively 
-
+```
+or, alternatively 
+```
```

Comment:
Don't you need `bash` here?

Reply:
To run "docker pull registry.connect.redhat.com/intel/openvino-model-server:latest" I first need to use registry token or login. Can you provide credentials for that?